app/src/mocks/userauth/login.ts

```diff
@@ -33,7 +33,7 @@ export const MOCK_USER = {
   username: "johndoe",
 };
 
-export const handlers = [
+export const loginMock = [
   http.get(/.*\.(png|jpg|jpeg|gif|svg)$/, async () => {
```

Comment:
1st handler is not part of the login mocking, it's a general one

Reply:
its okay i think i have a better structure for the mocks in general that we should adopt